### ISSUE TITLE
Itests failing on microk8s in main

### ISSUE THREAD

**Issue by PietroPasotti** [charmed-kubernetes/pytest-operator#77]
When I try to run the integration tests on main (microk8s), I get:
![image](https://user-images.githubusercontent.com/6230162/167575180-e60794a2-b605-4ae7-9b17-d4c53b4db213.png)

I noticed that the CI only has LXD env tests. Is it an idea to include microk8s to the substrates on which to run the itests?

To reproduce:
checkout main
tox -vve integration
wait for the boom

**Comment by stonepreston** [comment]
I think you need charm to build reactive charms, do the tests work after installing charm?
```
sudo snap install charm --classic
```

**Comment by addyess** [comment]
Agree, it's not an issue with microk8s, but an issue with not having the required tools installed to run the integration tests.

[charmed-kubernetes/actions-operator](https://github.com/charmed-kubernetes/actions-operator/blob/main/src/bootstrap/index.ts#L106)

I believe the integration tests test deploying a `machine bundle`(https://github.com/charmed-kubernetes/pytest-operator/blob/main/tests/integration/test_pytest_operator.py#L12-L47) into the model

Because of this, deploying into a kubernetes type model is yet to be supported for the integration tests.
